explainability-service/src/test/java/org/kie/trustyai/service/endpoints/metrics/MockMetricsStorage.java

```diff
@@ -0,0 +1,63 @@
+package org.kie.trustyai.service.endpoints.metrics;
+
+import java.nio.ByteBuffer;
+import java.util.Random;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Alternative;
+
+import org.kie.trustyai.service.data.exceptions.StorageReadException;
+import org.kie.trustyai.service.data.exceptions.StorageWriteException;
+import org.kie.trustyai.service.data.storage.Storage;
+
+import io.quarkus.test.Mock;
+
+@Mock
+@Alternative
+@ApplicationScoped
+public class MockMetricsStorage extends Storage {
```

Comment:
This looks an empty mock, have you considered [`InjectMocks` or similar](https://quarkus.io/blog/mocking/#using-quarkusmock) to have a similar behaviour without the need to declare this class?

Reply:
Removed